backend/routes/health.py

```
@@ -19,12 +19,61 @@
+            "content": {
+                "application/json": {
+                    "examples": {
+                        "healthy": {"summary": "Application running", "value": {"status": "healthy"}},
+                        "degraded": {"summary": "Application starting", "value": {"status": "degraded"}},
+                    }
+                }
+            },
+        }
+    },
+)
 async def root(state: dict = Depends(get_app_state)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)